### PR TITLE
Deprecate Bower package in favor of NPM

### PR DESCRIPTION
**Pull request by phndiaye**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  deploy:
```
Comment (AlexisMontagne) [inline .circleci/config.yml]: release rather than deploy?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version: 2
+  build-and-deploy:
+    jobs:
+      - deploy:
+          filters:
+            branches:
+              only:
+                - master
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
```
Comment (AlexisMontagne) [inline .circleci/config.yml]: /v[0-9]+(\.[0-9]+){2}/

Comment (tgallice) [inline .circleci/config.yml]: Are you sure this filters work ? IMO it's tags or branch filter

Comment (phndiaye) [reply .circleci/config.yml]: thanks

File: .circleci/config.yml (hunk 1 above)
Comment (phndiaye) [reply .circleci/config.yml]: yep ! better indeed

File: .circleci/config.yml (hunk 2 above)
Comment (tgallice) [reply .circleci/config.yml]: In ember-upf-utils its ignore all branch to be binded only on tags. It's not totally the same thing.

Comment (phndiaye) [reply .circleci/config.yml]: yes, i wanted to release only on master and if the commit matches a tag version. so we can't be in a branch and still be able to release (unless we should be able to do that?)

Comment (tgallice) [reply .circleci/config.yml]: I'm pretty sure you can't

**Pull request merged**